SCP -> BTP, Cloud Foundry - Cloud Foundry environment (#199)
SAP/cloud-sdk: docs/java/guides/cf-cli.mdx

---
id: cf-cli
title: SAP Cloud Foundry environment
hide_title: false
hide_table_of_contents: false

> sidebar_label: Cloud Foundry environment CLI

Reviewer: ```suggestion
sidebar_label: Cloud Foundry CLI
```

> description: Configure you Cloud Foundry environment CLI and bind it to SAP Business Technology Platform

Reviewer: ```suggestion
description: Configure you Cloud Foundry CLI and bind it to SAP Business Technology Platform
```

image:
---

> ## Cloud Foundry environment Command-Line Interface

Author: ```suggestion
## Cloud Foundry Environment Command-Line Interface
```

> ## Cloud Foundry Environment Command-Line Interface

Reviewer: ```suggestion
## Cloud Foundry Command-Line Interface
```

> To deploy your app developed with SAP Cloud SDK to [SAP Business Technology Platform](https://www.sap.com/products/cloud-platform.html) you'll need Cloud Foundry environment command-line interface (CLI).

Reviewer: ```suggestion
To deploy your app developed with SAP Cloud SDK to [SAP Business Technology Platform](https://www.sap.com/products/cloud-platform.html) you'll need Cloud Foundry command-line interface (CLI).
```

cf
```

> ## Bind Your Command-Line Interface To SAP Cloud Foundry environment

Author: ```suggestion
## Bind Your Command-Line Interface To SAP BTP Cloud Foundry environment
```

> ## Bind Your Command-Line Interface To SAP BTP Cloud Foundry environment

Reviewer: ```suggestion
## Bind Your Command-Line Interface to an API Endpoint
```

> Let's associate you Cloud Foundry environment CLI to your SAP account by providing an API endpoint and logging in with your account.

Reviewer: ```suggestion
Let's associate your Cloud Foundry CLI to your SAP account by providing an API endpoint and logging in with your account.
```

cf api https://api.cf.eu10.hana.ondemand.com
```

> Provide your credential for SAP Cloud Foundry environment by running:

Author: ```suggestion
Provide your credential for SAP BTP Cloud Foundry environment by running:
```

> Provide your credential for SAP BTP Cloud Foundry environment by running:

Reviewer: ```suggestion
Provide your credentials for SAP BTP:
```

```bash
cf login
```

> For more details on `SAP Cloud Foundry environment CLI` follow this [official tutorial](https://developers.sap.com/tutorials/cp-cf-download-cli.html).

Author: ```suggestion
For more details on `SAP BTP Cloud Foundry environment CLI` follow this [official tutorial](https://developers.sap.com/tutorials/cp-cf-download-cli.html).
```

> For more details on `SAP BTP Cloud Foundry environment CLI` follow this [official tutorial](https://developers.sap.com/tutorials/cp-cf-download-cli.html).

Reviewer: ```suggestion
For more details on the Cloud Foundry CLI and SAP BTP follow this [official tutorial](https://developers.sap.com/tutorials/cp-cf-download-cli.html).
```

## SAP Business Technology Platform

> Find out more about `SAP Business Technology Platform` and `Cloud Foundry environment` from [official documentation](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/73beb06e127f4e47b849aa95344aabe1.html).

Reviewer: ```suggestion
Find out more about the SAP Business Technology Platform, Cloud Foundry environment in the [official documentation](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/73beb06e127f4e47b849aa95344aabe1.html).
```
